services/web/server/tests/unit/isolated/test__json_schema.py

```diff
@@ -0,0 +1,35 @@
+import json
+from pathlib import Path
+
+from servicelib.aiohttp import jsonschema_validation
+from simcore_service_webserver._resources import resources
+
+
+def test_validate_project_json_schema():
+    CURRENT_DIR = Path(__file__).resolve().parent
+
+    with open(CURRENT_DIR / "data/project-data.json") as f:
+        project = json.load(f)
+
+    # NOTE: Mar.2023 Previous project-v0.0.1.json schema is passing validation
+    # even though it should fail. The reason is there is wrongly defined
+    # pattern properties in workbench which is ignored in practice.
+    # Current: "^[0-9a-fA-F]{8}-?[0-9a-fA-F]{4}-?4[0-9a-fA-F]{3}-?[89abAB][0-9a-fA-F]{3}-?[0-9a-fA-F]{12}$"
+    # Should be: "^[0-9a-fA-F]{8}-?[0-9a-fA-F]{4}-?[0-9a-fA-F]{4}-?[0-9a-fA-F]{4}-?[0-9a-fA-F]{12}$"
+    with resources.stream("api/v0/schemas/project-v0.0.1.json") as fh:
+        old_project_schema = json.load(fh)
+
+    jsonschema_validation.validate_instance(project, old_project_schema)
+
+    # This is new Pydantic generated json schema, that was modified
+    # so it works as the previous one. This should be fixed in:
+    # https://github.com/ITISFoundation/osparc-simcore/issues/3992
+    with resources.stream("api/v0/schemas/project-v0.0.1-pydantic.json") as fh:
+        new_project_schema = json.load(fh)
+
+    new_project_schema["properties"]["workbench"].pop("patternProperties")
+    new_project_schema["properties"]["ui"]["properties"]["workbench"].pop(
+        "patternProperties"
+    )
```

Comment:
So I'd guess, you expect the `patternProperties` key to be present in those objects. Could you explicitly assert for them in the object?

Doing so you can add a note on how to change the code back, when Pydantic fixes the issue. With notes on removing these `pop` actions from the codebase.